packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -2509,7 +2509,7 @@ function pushStyleImpl(
     target.push(stringToChunk(escapeTextForBrowser('' + child)));
   }
   pushInnerHTML(target, innerHTML, children);
-  target.push(endTag1, stringToChunk('style'), endTag2);
+  target.push(endChunkForTag('style'));
```

Comment:
might as well precompute these

Reply:
not well, it is already cached so the computation that causes the memory allocation is only executed once

Reply:
@acdlite